ledger/src/blockstore_db.rs

```diff
@@ -293,62 +293,60 @@ impl OldestSlot {
 }
 
 #[derive(Debug)]
-struct Rocks(rocksdb::DB, ActualAccessType, OldestSlot);
+struct Rocks(rocksdb::DB, AccessType, OldestSlot);
 
 impl Rocks {
     fn open(path: &Path, options: BlockstoreOptions) -> Result<Rocks> {
-        let access_type = &options.access_type;
+        let access_type = options.access_type.clone();
```

Comment:
Yep, this clone is necessary so as it to keep `options` whole such that it can be borrowed by `cf_descriptors()` (partial moves are a thing but partial borrows are not).